cpp/core.cc

```diff
@@ -254,6 +254,27 @@ bool InputAvailable(int fd) {
   return select(FD_SETSIZE, &fds, NULL, NULL, &timeout) > 0;
 }
 
+List<int>* WaitForInputs(List<int>* fds_in) {
+  auto* ret = NewList<int>();
+  fd_set fds;
+  FD_ZERO(&fds);
```

Comment:
`fds` could be something like `fd_bits`, to distinguish from `fd_list`

It is a bit mask passed to the kernel